chess/svg.py

```diff
@@ -177,6 +178,7 @@ def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=No
             "class": " ".join(cls),
             "stroke": "none",
             "fill": fill_color,
+            "class": "square",
```

Comment:
This would override the existing class, which already contains square (and possibly dark or light and/or lastmove) anyway.